### PR TITLE
[DOCS] Fix default value for closed indices

### LINKED ISSUE #78877: [DOCS] `cluster.indices.close.enable` default is incorrect
The `cluster.indices.close.enable` cluster setting defaults to `true`. The [index management setting docs][0] incorrectly list the default as `false`.

Relevant code:
https://github.com/elastic/elasticsearch/blob/ad9760325c337087e76d8ed98961c2a635923d70/server/src/main/java/org/elasticsearch/action/admin/indices/close/TransportCloseIndexAction.java#L46-L47

The setting also relates to closing indices. You can open closed indices even when the setting is `false`.

[0]: https://www.elastic.co/guide/en/elasticsearch/reference/master/index-management-settings.html#cluster-indices-close-enable

Relates to https://github.com/elastic/elasticsearch/pull/57953

### PR DESCRIPTION
**Pull request by lockewritesdocs**
#57953 introduced changes that added ESS icons to many Elasticsearch settings. As part of those changes, the default value for `cluster.indices.close.enable` was indicated as `false`, when it should be `true`. This PR updates the default value to `true`. 

Closes #78877

### CODE REVIEW

**Review by jrodewig — COMMENTED** [review]
Thanks for updating this @lockewritesdocs. I think an engineer should take a look before we merge. In particular, I think the `IMPORTANT` admon may be outdated.

**Review by AthenaEryma — APPROVED** [review]
LGTM - thanks!

**Pull request merged**
